packages/@aws-cdk/aws-dynamodb/test/dynamodb.test.ts

```diff
@@ -719,6 +721,70 @@ test('if an encryption key is included, encrypt/decrypt permissions are added to
   });
 });
 
+test('when specifying STANDARD_INFREQUENT_ACCESS table class', () => {
+  const stack = new Stack();
+  new Table(stack, CONSTRUCT_NAME, {
+    tableName: TABLE_NAME,
+    partitionKey: TABLE_PARTITION_KEY,
+    tableClass: TableClass.STANDARD_INFREQUENT_ACCESS,
+  });
+
+  Template.fromStack(stack).hasResourceProperties('AWS::DynamoDB::Table',
+    {
+      KeySchema: [
+        { AttributeName: 'hashKey', KeyType: 'HASH' },
+      ],
+      TableClass: 'STANDARD_INFREQUENT_ACCESS',
+      AttributeDefinitions: [
+        { AttributeName: 'hashKey', AttributeType: 'S' },
+      ],
+      TableName: 'MyTable',
+    },
+  );
+});
+
+test('when specifying STANDARD table class', () => {
+  const stack = new Stack();
+  new Table(stack, CONSTRUCT_NAME, {
+    tableName: TABLE_NAME,
+    partitionKey: TABLE_PARTITION_KEY,
+    tableClass: TableClass.STANDARD,
+  });
+
+  Template.fromStack(stack).hasResourceProperties('AWS::DynamoDB::Table',
+    {
+      KeySchema: [
+        { AttributeName: 'hashKey', KeyType: 'HASH' },
+      ],
+      TableClass: 'STANDARD',
+      AttributeDefinitions: [
+        { AttributeName: 'hashKey', AttributeType: 'S' },
+      ],
+      TableName: 'MyTable',
+    },
+  );
+});
+
+test('when specifying no table class', () => {
+  const stack = new Stack();
+  new Table(stack, CONSTRUCT_NAME, {
+    tableName: TABLE_NAME,
+    partitionKey: TABLE_PARTITION_KEY,
+  });
+
+  Template.fromStack(stack).hasResourceProperties('AWS::DynamoDB::Table',
```

Comment:
In this test, we should also verify that a `TableClass` property is not present in the resulting template:

```ts
  Template.fromStack(stack).hasResourceProperties('AWS::DynamoDB::Table', Match.not(
    {
      TableClass: Match.anyValue(),
    }),
  );
```

Reply:
Good point, however I did implement it slightly different with Match.absent().